### PR TITLE
Fixes an issue with networks unindexable by name

### PR DESCRIPTION
**Pull request by blurpesec**
### Description
ethers.js allows you to add networks using network id for some networks, but not for networks that are unsupported. Noticed this issue first while looking into #3532.

### CODE REVIEW

File: src/services/EthService/network/helpers.ts [hunk 1]
```diff
@@ -7,51 +7,61 @@ import { ETHERSCAN_API_KEY, INFURA_API_KEY } from '@config';
 import { DPathFormat, Network, NetworkId, NodeOptions, NodeType } from '@types';
 
 // Network names accepted by ethers.EtherscanProvider
-type TValidEtherscanNetwork = 'homestead' | 'ropsten' | 'rinkeby' | 'kovan' | 'goerli';
+type TValidEthersNetworkish = 'homestead' | 'ropsten' | 'rinkeby' | 'kovan' | 'goerli' | number;
```
Comment (vilsbole) [inline src/services/EthService/network/helpers.ts:10]: What does `networkish` refer to?!

**Review by vilsbole — CHANGES_REQUESTED** [review]
Include tests for modified code.

File: src/services/EthService/network/helpers.ts (hunk 1 above)
Comment (blurpesec) [reply src/services/EthService/network/helpers.ts:10]: > What does `networkish` refer to?!

When setting a network using ethers.js they have string config specified for the above networks. But in the instance where it doesn't have a config for the network name, you need to use a chainid. This input type is called [Networkish](https://docs.ethers.io/v5/api/providers/types/#providers-Networkish) for some reason.

**Review by vilsbole — COMMENTED** [review]
Code lgtm

**Review by vilsbole — APPROVED** [review]

**Pull request merged**